test/src/test/java/org/axonframework/test/aggregate/FixtureTest_Annotated.java

```diff
@@ -37,17 +39,21 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
 
 /**
  * @author Allard Buijze
  */
 public class FixtureTest_Annotated {
 
     private FixtureConfiguration<AnnotatedAggregate> fixture;
+    private CommandTargetResolver mockCommandTargetResolver;
 
     @Before
     public void setUp() {
         fixture = new AggregateTestFixture<>(AnnotatedAggregate.class);
+        mockCommandTargetResolver = mock(CommandTargetResolver.class);
+
```

Comment:
Super nit: being nit-picky here, but this empty line is not necessary I think.

Reply:
Done.